src/cmdlib.sh

```diff
@@ -210,6 +216,16 @@ prepare_build() {
             # level to avoid burning excessive CPU.
             ostree --repo="${tmprepo}" config set archive.zlib-level 2
         fi
+
+        # No need to fsync for transient flows
+        if is_transient; then
+            ostree --repo="${tmprepo}" config set 'core.fsync' 'false'
+        fi
+
+        # No need to fsync for transient flows
+        if test -f "${workdir}/tmp/cosa-transient"; then
+            ostree --repo="${tmprepo}" config set 'core.fsync' 'false'
+        fi
```

Comment:
This is doing the same thing twice.

Reply:
Oops, merge conflict resolution failure; fixed